hack/env/deploy-latest-kubevirt.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+NAMESPACE=${1:-kubevirt}
+
+oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${NAMESPACE}
+EOF
+
+# Deploying kuebvirt
+LATEST_KUBEVIRT=$(curl -L https://api.github.com/repos/kubevirt/kubevirt/releases | \
```

Comment:
Why are you fetching latest kubevirt version, but later you are using CR from v0.35.0?

Reply:
no specific reason, just copied this from `ssp-operator`

Reply:
Maybe you could fetch the latest released CR and modify it according to what we need with `jq`, I won't press on it because if the KV CR API ever changes, we would have to change this code as well, so I am fine with both approaches (hard-coded CR definition or dynamically fetching it from the latest release)

Reply:
Should't we stay with kubevirt version, which was released when the validator was deprecated? Because since this repo will not receive any new update, I think it would be good to stick with stable old version.

Reply:
Yes, you are right Karel, that point slipped my mind, we should stay with the fixed version of kubevirt in which this repo was depricated